Rakefile

```diff
@@ -11,11 +11,25 @@ require 'colorize'
 require 'highline'
 require 'tmpdir'
 require 'json'
+require 'zip'
 
 Cucumber::Rake::Task.new(:features) do |t|
   t.cucumber_opts = "--format pretty"
 end
 
+desc("Updates swift-doc binary with the latest version available.")
+task :swift_doc_update do
+  version = "1.0.0-beta.4"
+  system("curl", "-LO", "https://github.com/SwiftDocOrg/swift-doc/archive/#{version}.zip")
+  extract_zip("#{version}.zip", "swift-doc")
+  Dir.chdir("swift-doc/swift-doc-#{version}") do
+    system("make", "swift-doc")
+  end
+  system("cp", "swift-doc/swift-doc-#{version}/.build/release/swift-doc", "vendor/swift-doc")
```

Comment:
I'd recommend creating a file alongside the binary that indicates its version. We can call it `swift-doc.version`.